main.py

```diff
@@ -1,10 +1,41 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as json_config:
+        config = json.load(json_config)
+
+    for username, user_data in config.items():
+
+        race = Race.objects.get_or_create(
+            name=user_data["race"]["name"],
+            description=user_data["race"]["description"]
+        )[0]
```

Comment:
Use unpacking instead of indexing here and in the other places as well
```suggestion
        race, _ = Race.objects.get_or_create(
            name=user_data["race"]["name"],
            description=user_data["race"]["description"]
        )
```

Reply:
It only worked with 'race '. 
With 'guild', I get the 'TypeError: cannot unpack non-iterable NoneType object' error